content/id/guides/components-glossary/pages-validate.md

```diff
@@ -0,0 +1,70 @@
+---
+title: 'Metode Validasi'
+description: Nuxt.js memungkinkan Anda menentukan metode validator di dalam komponen rute dinamis Anda.
+menu: Metode Validasi
+category: komponen-glosarium
+---
+
+> Nuxt.js memungkinkan Anda menentukan metode validator di dalam komponen rute dinamis Anda.
+
+- **Type:** `Function` atau `Async Function`
+
+`validate` dipanggil setiap kali sebelum menavigasi ke rute baru. Ini akan disebut sisi server sekali (pada permintaan pertama ke aplikasi Nuxt) dan sisi klien saat menavigasi ke rute lebih lanjut. Metode ini mengambil [`konteks`](/guides/internals-glossary/context) object as an argument.
+
+```js
+validate({ params, query, store }) {
+  return true // if the params are valid
+  return false // will stop Nuxt.js to render the route and display the error page
+}
+```
+
+```js
+async validate({ params, query, store }) {
+  // await operations
+  return true // if the params are valid
+  return false // will stop Nuxt.js to render the route and display the error page
+}
+```
+
+Anda juga dapat mengembalikan izin:
```

Comment:
`mengembalikan izin` ini konteks bahasa Inggrisnya apakah?

Reply:
maksudnya mas @jefrydco ? 
ini bahasa inggrisnya mas, `You can also return promises:`

Reply:
Jadi bagaimana yang benar kata kata nya mas @jefrydco ?

Reply:
promise sepertinya ngga usah diterjemahkan Mas